Bug fixes

tapabratadey opened the pull request.

tapabratadey committed:
1. Bug fixes homepage

Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>
Co-authored-by: marri-kang <marri-kang@users.noreply.github.com>
Co-authored-by: Summer Tinio <SummerTinio@users.noreply.github.com>

tapabratadey committed:
Bug fixes discover page

Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>
Co-authored-by: marri-kang <marri-kang@users.noreply.github.com>

tapabratadey committed:
Bug fixes

Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>
Co-authored-by: marri-kang <marri-kang@users.noreply.github.com>

sheikhrachel commented on src/backend/config/db.ts:
```diff
@@ -7,7 +7,8 @@ import { Like } from '../entities/Like';
 
 import chalk from 'chalk';
-
+/**
```
remove commented code

tapabratadey replied:
removed

sheikhrachel commented on src/backend/controller/comment/comment.ts:
```diff
@@ -1,25 +1,28 @@
 const commentController = {
   getAllById: async (req: Request) => {
-    return commentService.getAllById(req.body, req.query);
+    console.log('in the controller');
```
remove log

tapabratadey replied:
removed

sheikhrachel commented on src/backend/controller/project/ProjectController.ts:
```diff
@@ -50,8 +55,10 @@ const ProjectController = {
   findById: async (req) => {
     try {
+      const { id } = req.query;
+      console.log(chalk.green('@line 59: ', id));
```
remove log

tapabratadey replied:
removed

sheikhrachel commented on src/backend/services/db/comment/commentService.ts:
```diff
@@ -4,45 +4,48 @@ import { Comment } from '@backend/entities/Comment';
-    catch {
-        throw new DatabaseError(dbError);
+      const db = await connection();
+      console.log(chalk.green(getConnection().isConnected));
```
remove log

tapabratadey replied:
removed

sheikhrachel commented on src/backend/services/db/comment/commentService.ts:
```diff
@@ -4,45 +4,48 @@ import { Comment } from '@backend/entities/Comment';
     const id = query.id;
-    const db = await connection();
-
+    console.log(chalk.green(id));
```
remove log

tapabratadey replied:
removed

sheikhrachel commented on src/backend/services/db/comment/commentService.ts:
```diff
@@ -4,45 +4,48 @@ import { Comment } from '@backend/entities/Comment';
+        .innerJoin('user', 'user', 'comment.userId = user.id')
+        .where('"projectId" = :id', { id })
+        .getRawMany();
+      console.log(comments);
```
remove log

sheikhrachel commented on src/backend/services/db/comment/commentService.ts:
```diff
@@ -4,45 +4,48 @@ import { Comment } from '@backend/entities/Comment';
+      console.log(comments);
+      return comments;
+    } catch (err) {
+      console.log(chalk.red('================='), err);
```
remove log

sheikhrachel commented on src/backend/services/db/project/ProjectService.ts:
```diff
@@ -73,12 +73,19 @@ const ProjectService = {
           `project.user = (SELECT id FROM public.user WHERE uid = '${uid}')`
         )
         .getRawMany();
+      console.log(userData);
```
remove log

tapabratadey replied:
removed

sheikhrachel commented on src/backend/services/db/project/ProjectService.ts:
```diff
@@ -114,21 +134,23 @@ const ProjectService = {
+        .where(`project.id = :id`, { id })
         .getRawOne();
-
+      console.log('================', foundProject);
```
remove log

tapabratadey replied:
removed

sheikhrachel commented on src/backend/services/db/project/ProjectService.ts:
```diff
@@ -114,21 +134,23 @@ const ProjectService = {
         throw new DatabaseError(dbError);
       }
-
+      console.log(chalk.green(getConnection().isConnected));
```
remove log

tapabratadey replied:
removed

sheikhrachel commented on src/backend/services/db/project/ProjectService.ts:
```diff
@@ -138,16 +160,17 @@ const ProjectService = {
       }
       return [foundProject, StatusCodes.OK];
     } catch (err) {
+      console.log(chalk.red('================= @Line 144: '), err);
```
remove log

tapabratadey replied:
removed

sheikhrachel commented on src/backend/services/db/project/ProjectService.ts:
```diff
@@ -320,16 +343,20 @@ const ProjectService = {
+    .innerJoin(User, 'user', 'project.userId = user.id')
+    .orderBy(`project.${column}`, 'DESC')
+    .getRawMany();
+  //console.log(projectData);
```
remove log

tapabratadey replied:
removed

tapabratadey committed:
Refactor db.ts
1. Removing commented code
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Removing logs
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Update recommendationService.ts
- Add pictures from return projects
- Update return length to 5
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
JumpStarter home page image

tapabratadey committed:
Update db.ts
1. Remove Logging is true from options
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Removing logs
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Removing logs
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Removing logs
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Update userService.ts
- Querying user pictures
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Rendering Landing Page Image
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Update largeprojectcard.tsx
- Remove comments
- Adding firstname/lastname types
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Update sectionMarquee.tsx
- Remove comments
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Update singleProjectInfo.tsx
- Add props.likesAmt to useEffect and add to deps array
- Add onclick method
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Update urls.ts
- Adding URL constants
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Update discover.tsx
- Sorting works
- Refactor code
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Update Homepage
- Refactor code
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
DELETE file

tapabratadey committed:
Update Homepage
- Remove logs
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Update Recommend.ts
- Remove logs
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
DELETE file

tapabratadey committed:
Update Recommend.ts
- Refactor code
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Update Recommend.ts
- Refactor code
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Update Profile page
- Refactor code
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Update Project by ID page
- Render Project Launch Date
- Render user avatar for comments
- Refactor code
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Merge branch 'main' into bug-fixes

Co-authored-by: marri-kang <marri-kang@users.noreply.github.com>
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Updates
1. Fix filter
2. Fix UI/UX

Co-authored-by: marri-kang <marri-kang@users.noreply.github.com>
Co-authored-by: Pranitha Maganty <pmaganty@users.noreply.github.com>

tapabratadey committed:
Uncomment motion.div
